src/jvm/test/org/jetbrains/kotlinx/lincheck_test/representation/ThreadCreationRepresentationTest.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Lincheck
+ *
+ * Copyright (C) 2019 - 2025 JetBrains s.r.o.
+ *
+ * This Source Code Form is subject to the terms of the
+ * Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+ * with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package org.jetbrains.kotlinx.lincheck_test.representation
+
+import kotlin.concurrent.thread
+
+class ThreadCreationRepresentationTest: BaseTraceRepresentationTest("thread_creation_representation_test.txt") {
```

Comment:
Please use `BaseRunConcurrentRepresentationTest` as a base for new tests. 
It runs the code in GPMC mode and does not involve Lincheck's data structures testing functionality.